stof/doctrine-extensions-bundle/1.2/etc/packages/stof_doctrine_extensions.yaml

```diff
@@ -0,0 +1,21 @@
+stof_doctrine_extensions:
+    default_locale: en_US
+
+    # Activate the extensions you want
+    orm:
+        default:
+            tree: true
+            timestampable: false # not needed: listeners are not enabled by default
+        other:
```

Comment:
It is done, I simplified by adding the links of documentation which is too consequent to be expressed here.